components/match2/wikis/apexlegends/summary_util.lua

```diff
@@ -0,0 +1,208 @@
+---
+-- @Liquipedia
+-- wiki=apexlegends
+-- page=Module:Summary/Util
```

Comment:
not 100% sold on the name
like summary seems generic could be a tournament summary instead of game/match summary

Reply:
Open for suggestions :)

Reply:
clunky version would be something like MatchOrGameSummary/Util
but don't like that either
can't come up with a better name, still not ideal though
we can rename if the need arises later on :)